.github/workflows/build.yml

```diff
@@ -24,6 +24,16 @@ jobs:
     - name: build
       run: |
            python3 ci/build-in-all-configs.py --${{matrix.compiler}} --default-config
+  build-linux-bazel:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v4
+    - name: bazel build opt
+      run: |
+           bazel build //... -c opt
+    - name: bazel build dbg
+      run: |
+           bazel build //... -c opt
```

Comment:
Is it running the same command twice?

Reply:
Oh this is typo. I was thinking to build cpptrace in both debug and release mode. Or do you think only one of them is enough?

Reply:
Either way is fine with me, it doesn't hurt to do both